CONTRIBUTING.md

```diff
@@ -30,6 +30,17 @@ When filing an issue, please do *NOT* include:
 - Internal identifiers such as JIRA tickets
 - Any sensitive information related to your environment, users, etc.
 
+## Documentation
+
+The Splunk Observability documentation is hosted on https://docs.splunk.com/Observability,
```

Comment:
```suggestion
The Splunk Observability documentation is hosted at https://docs.splunk.com/Observability,
```

Reply:
Quite the debate, interestingly: https://english.stackexchange.com/questions/8226/on-website-or-at-website

Reply:
lol